lib/CL/devices/spirv_queries.h

```diff
@@ -0,0 +1,42 @@
+/* spirv_queries.h - sets up SPIR-V queries for the device
+ *
+ * Copyright (c) 2025 Ben Ashbaugh
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+#ifndef SPIRV_QUERIES_H
+#define SPIRV_QUERIES_H
+
+#include "devices.h"
+#include "pocl_export.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+  POCL_EXPORT
```

Comment:
Can you indent this back? The formatter script [needs updating](https://github.com/pocl/pocl/pull/1942#discussion_r2126580463).